cves/CVE-2017-5650.yml

```diff
@@ -44,6 +44,11 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description:
+  This CVE describes a Denial of Service attack done through the network via malicious HTTP requests.
+  More specifically, the flaw can be encountered in the handling of a HTTP/2 GOAWAY frame for a connection did not
+  close its associated streams that were waiting for a WINDOW_UPDATE before allowing the application to write more data.
+  These waiting streams each consumed a thread. Therefore, an attacker could generate a DoS attack through a malicious client
+  that constructed a series of HTTP/2 requests that would consume all available processing threads.
```

Comment:
There is a lot of jargon here that makes this hard to read. Things like "HTTP/2 GOAWAY frame" and "WINDOW_UPDATE" make this hard to understand for someone with only general programming knowledge, who this description should be written for.

Reply:
Gave a more elaborate description.